src/Pages/Playlist/Playlist.js

```diff
@@ -1,7 +1,58 @@
+import { usePlaylist } from '../../Context/PlaylistContext'
+import './Playlist.css'
+import { Link, useNavigate } from 'react-router-dom'
+import { useAuth } from '../../Context/AuthContext'
 export const Playlist = () =>{
+    const {isLogin} = useAuth()
+    const navigate = useNavigate()
+    const { playlist, deletePlaylist } = usePlaylist()
+    
     return(
-        <>
-            Playlist page
-        </>
+        <div className='playlist-container'>
+            {isLogin ? 
+                playlist.length>0 ? <>
+                {
+                    playlist.map((item) => 
+                        <div className='playlist'>
```

Comment:
Suggestion: use key attribute whenever you are using map method for rendering.

Reply:
Sure I forgot it every single time